SWTableViewCell/PodFiles/SWTableViewCell.m

```diff
@@ -676,8 +725,13 @@ - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoi
     {
         CGFloat leftThreshold = [self contentOffsetForCellState:kCellStateLeft].x + (self.leftUtilityButtonsWidth / 2);
         CGFloat rightThreshold = [self contentOffsetForCellState:kCellStateRight].x - (self.rightUtilityButtonsWidth / 2);
-        
-        if (targetContentOffset->x > rightThreshold)
+        CGFloat rightLongSwipeTriggerThreshold = [self offsetForRightLongSwipeTrigger];
+
+        if (targetContentOffset->x > rightLongSwipeTriggerThreshold)
```

Comment:
targetContentOffset is an inout parameter. This means the parameter can be modified - it's a little strange, since this is more of a C convention, but the API is such.